modules/services/emacs.nix

```diff
@@ -101,13 +101,27 @@ in {
         editor using the <envar>EDITOR</envar> environment variable.
       '';
     };
+
+    afterGraphicalSessionTarget = mkOption {
```

Comment:
Instead of a new option, we should extend the type of `startWithUserSession` from `bool` to something like `enum [ false true "graphical" ]`.